go/kbfs/libkbfs/folder_branch_status.go

```diff
@@ -48,6 +48,8 @@ type FolderBranchStatus struct {
 	Unmerged []*crChainSummary
 	Merged   []*crChainSummary
 
+	ConflictResolutionAttempts []conflictRecord
```

Comment:
Please tag with `json:",omitempty"` so it doesn't show up when there are no records.